libbpf-rs/src/program.rs

```diff
@@ -624,6 +624,30 @@ impl<'obj> Program<'obj> {
         Ok(prog_info.id)
     }
 
+    /// Returns program fd by pin
+    pub fn get_fd_by_pin<P: AsRef<Path>>(path: P) -> Result<OwnedFd> {
```

Comment:
I understand why you named the method this way but the existing `..._by_...` naming is horrible, in my opinion. What do you think about `fd_from_pinned_path`, analogous to https://docs.rs/libbpf-rs/0.24.2/libbpf_rs/struct.MapHandle.html#method.from_pinned_path?

Reply:
Renamed as proposed. I think `..._by_...` was somehow influenced by `bpf.h`, and I agree, `from` is more natural here.